plugins/SegmentEditor/tests/Integration/SegmentEditorTest.php

```diff
@@ -145,7 +147,7 @@ public function testUpdateSegment()
         $this->clearReArchiveList();
 
         $updatedSegment = array(
-            'idsegment' => $idSegment2,
+            'idsegment' => '' . $idSegment2,
```

Comment:
Is this meant to convert the variable to string? Might be better to explicitly type it using

```suggestion
            'idsegment' => (string) $idSegment2,
```

Reply:
✅ Done!